src/styles/utils/variable.scss

```
@@ -0,0 +1,6 @@
+$color-main: #060b35;
+$color-secondary: #616070;
+$button-color-blue: #00acdc;
+$button-color-white: #fff;
```

It is better to use a more universal naming.

```suggestion
$button-color: #00acdc;
$button-color-active: #fff;
```